tests/other/metadce/mem_no_args.c

```diff
@@ -0,0 +1,7 @@
+#include <stdlib.h>
```

Comment:
I would call this `no_argv` so since is more obvious what args we are dealing with.

Reply:
Sure, renaming.